### PR TITLE
Rewrite MovingTool to custom solution

### PR DESCRIPTION
**Pull request by piotrpekala7**

### CODE REVIEW

File: src/app/cartography/components/d3-map/d3-map.component.ts [hunk 1]
```diff
@@ -121,7 +121,7 @@ export class D3MapComponent implements OnInit, OnChanges, OnDestroy {
 
     this.subscriptions.push(
       this.toolsService.isMovingToolActivated.subscribe((value: boolean) => {
-        this.movingToolWidget.setEnabled(value);
+        //this.movingToolWidget.setEnabled(value);
```
Comment (ziajka) [inline src/app/cartography/components/d3-map/d3-map.component.ts]: Remove

File: src/app/cartography/directives/zooming-canvas.directive.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        });
+    }
+
+    ngOnDestroy() {
+        this.movingModeState.unsubscribe();
+    }
+
+    @HostListener('window:wheel', ['$event'])
+    onWheel(event: WheelEvent) {
+        if (this.activated) {
```
Comment (ziajka) [inline src/app/cartography/directives/zooming-canvas.directive.ts]: Reverse condition.

Comment (piotrpekala7) [reply src/app/cartography/directives/zooming-canvas.directive.ts]: Cannot be applied

**Comment by piotrpekala7** [comment]
Due to https://github.com/facebook/react/issues/14856 we should reimplement zooming directive

**Pull request merged**